perllib/FixMyStreet/Cobrand.pm

```diff
@@ -159,7 +159,14 @@ sub body_handler {
     foreach my $avail ( $class->available_cobrand_classes ) {
         my $cobrand = $class->get_class_for_moniker($avail->{moniker})->new({});
         next unless $cobrand->can('council_area_id');
-        return $cobrand if $areas->{$cobrand->council_area_id};
+
+        # The majority of cobrands only cover a single area, but e.g. Northamptonshire
```

Comment:
All of this would I think be fixed/simplified by the suggestion we discussed of assigning a cobrand to a body, I think. As it is, if we want to do this step first, I think we'll need some new tests given none of this is caught by anything; I think the fix for now will have to be an exemption in Body's get_cobrand_handler similar to get_body_handler_for_problem, maybe?